src/ui/components/common/nav.rs

```diff
@@ -1,213 +1,151 @@
-                                        <hr/>
-                                      </li>
-                                      <li>
-                                        <ActionForm action=logout_form_action>
-                                          <input
-                                            name="is_ssr"
-                                            type="hidden"
-                                            value=format!("{}", is_ssr.get())
-                                          />
-                                          <button type="submit">{t!(i18n, nav.logout)}</button>
-                                        </ActionForm>
-                                      </li>
-                                    </ul>
-                                  </details>
-                                </li>
-                              }
-                          }
-                      })
-              }}
+            >
 
-            </ErrorBoundary>
-          </Suspense>
+              <li>
+                <A href="/inbox">
+                  <span title=t!(i18n, nav.unread_messages)>
+                    <Icon icon=Icon::from(ChIcon::ChBell) class="h-6 w-6"/>
+                  </span>
+                </A>
+              </li>
+              <li>
+                <details>
+                  <summary>
+                    {with!(
+                        | my_user | { let Person { name, display_name, .. } = my_user.as_ref()
+                        .unwrap(); display_name.as_ref().unwrap_or(name).to_string() }
+                    )}
 
-        </ul>
-      </div>
-    </nav>
+                  </summary>
+                  <ul>
+                    <li>
+                      <A href=with!(
+                          | my_user | format!("/u/{}", my_user.as_ref().unwrap().name)
+                      )>{t!(i18n, nav.profile)}</A>
+                    </li>
+                    <li>
+                      <A href="/settings">{t!(i18n, nav.settings)}</A>
+                    </li>
+                    <li>
+                      <hr/>
+                    </li>
+                    <li>
+                      <ActionForm action=logout_action>
+                        <button type="submit">{t!(i18n, nav.logout)}</button>
+                      </ActionForm>
+                    </li>
+                  </ul>
+                </details>
+              </li>
+            </Show>
+          </ul>
+        </div>
+      </nav>
+    </Transition>
   }
 }
 
 #[component]
 pub fn BottomNav() -> impl IntoView {
   let i18n = use_i18n();
-
   view! {
     <nav class="container navbar mx-auto">
       <div class="navbar-start"></div>
       <div class="navbar-end ">
         <ul class="menu menu-horizontal flex-nowrap">
           <li>
-            <a href="//github.com/LemmyNet/lemmy-ui-leptos/releases" class="text-md">
+            <a href="github.com/LemmyNet/lemmy-ui-leptos/releases" class="text-md">
```

Comment:
i think the double slash is needed for the url to be absolute.

Reply:
You're right. Added them back in.